### PR TITLE
Implement 3 missing AC setting methods

### PR DESCRIPTION
**Pull request by prvakt**
- set_ac_at_unlock (enable/disable AC at unlock)	
- set_seats_heating (enable/disable seats[frontLeft,frontRight] heating with AC)
- set_windows_heating (enable/disable windows heating with AC)

### CODE REVIEW

File: myskoda/rest_api.py [hunk 1]
```diff
@@ -327,6 +327,33 @@ async def set_ac_without_external_power(self, vin: str, enabled: bool) -> None:
...
+        json_data = {"airConditioningAtUnlockEnabled": "True" if enabled else "False"}
+        await self._make_post_request(
+            url=f"/v2/air-conditioning/{vin}/settings/ac-at-unlock",
+            json=json_data,
+        )
+
+    async def set_windows_heating(self, vin: str, enabled: bool) -> None:
+        """Enable or disable windows heating with AC."""
+        _LOGGER.debug("Setting windows heating with AC for vehicle %s to %r", vin, enabled)
+        json_data = {"windowHeatingEnabled": "True" if enabled else "False"}
```
Comment (WebSpider) [inline myskoda/rest_api.py:342]: Why does the seat-heating message have a proper class, but window-heating doesnt?

File: myskoda/rest_api.py [hunk 2]
```diff
@@ -327,6 +327,33 @@ async def set_ac_without_external_power(self, vin: str, enabled: bool) -> None:
             json=json_data,
         )
 
+    async def set_ac_at_unlock(self, vin: str, enabled: bool) -> None:
+        """Enable or disable AC at unlock."""
+        _LOGGER.debug("Setting AC at at unlock for vehicle %s to %r", vin, enabled)
+        json_data = {"airConditioningAtUnlockEnabled": "True" if enabled else "False"}
```
Comment (WebSpider) [inline myskoda/rest_api.py:333]: Why does the seat-heating message have a proper class, but ac-unlock doesnt?

**Review by WebSpider — COMMENTED** [review]
Thanks for this awesome addition! 👍

Some questions about consistency, but this looks like solid code 👍

**Comment by prvakt** [comment]
> Thanks for this awesome addition! 👍
> 
> Some questions about consistency, but this looks like solid code 👍

request for seats heating can contain one or more seats so its needs to be handled in more special way (like the auxiliary heater), but windows_heating or ac_at_unlock can be only activated or deactivated using always same parameter so I did not wanted to complicate it too much.. But of course I can modify it so its handled same as seats_heating

**Comment by WebSpider** [comment]
I do prefer using the same structure, even if it looks like some overhead. Python will optimize it, and it produces more predictable code

**Pull request merged**